### PR TITLE
Add feature for copying a permalink

### LINKED ISSUE #645: Expose a permalink URL
When pasting link to wpt.fyi into bug trackers as evidence of some problem, it would be very nice to paste links that will remain stable. When visiting that link, the ability to see the same results but for the latest runs would be great too.

I have manually been adding `sha=12345` when I wanted this myself.

### PR DESCRIPTION
**Pull request by lukebjerring**
## Description
Fixes #645 

Adds a small "Link" button next to "Edit" in the query banner, which opens a dialog allowing the user to tweak the URL a bit and copy to the clipboard with a click (in Chrome/FF, at least).

## Review Information
- Head to /flags, enable the permalink flag
- Visit the homepage, nav to a directory somewhere
- Hit edit, toggle the tickboxes, copy to clipboard

### CODE REVIEW

**Comment by foolip** [comment]
@lukebjerring no staging instance was deployed for this, a bug?

File: webapp/components/wpt-permalinks.js [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        paper-tabs {
+          --paper-tabs-selection-bar-color: var(--paper-blue-500);
+        }
+        paper-tab {
+          --paper-tab-ink: var(--paper-blue-300);
+        }
+      </style>
+      <paper-dialog>
+        <paper-tabs selected="{{selectedTab}}">
+          <paper-tab title="Link to these specific runs, via their IDs">These runs</paper-tab>
```
Comment (foolip) [inline webapp/components/wpt-permalinks.js]: Nice!

File: webapp/components/wpt-permalinks.js [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    return url;
+  }
+
+  async handleCopyToClipboard() {
+    try {
+      const input = this
+        .shadowRoot.querySelector('paper-input')
+        .shadowRoot.querySelector('input');
+      input.select();
+      document.execCommand('copy');
```
Comment (foolip) [inline webapp/components/wpt-permalinks.js]: This returns false when it doesn't work, it doesn't throw. That means the success toast will always show and the failure never. I haven't tested in Safari, but are you sure it works?

**Comment by lukebjerring** [comment]
Yeah we temporarily broke staging deployment in #1037 - should be fixed with the master-merge push.

**Comment by lukebjerring** [comment]
cc @Hexcles

**Comment by lukebjerring** [comment]
https://permalink-dot-wptdashboard-staging.appspot.com

**Comment by lukebjerring** [comment]
Works on safari 12 (tested on browser stack)

**Pull request merged**